com.unity.shadergraph/Editor/Data/Graphs/SamplerStateMaterialSlot.cs

```diff
@@ -30,6 +30,10 @@ internal override bool bareResource
             set { m_BareResource = value; }
         }
 
+        // NOT serialized -- this is always set by the parent node if they care about it
+        public TextureSamplerState defaultSamplerState { get; set; }
+        public string defaultSamplerStateName => defaultSamplerState?.defaultPropertyName ?? "SamplerState_Linear_Repeat";
```

Comment:
SubGraph nodes will always set up the defaultSamplerState on samplerstate slots when they UpdateSlots